case/src/main/java/com/ritense/document/domain/Document.java

```diff
@@ -56,7 +56,7 @@ public interface Document {
     @JsonProperty
     String assigneeFullName();
 
-    @JsonProperty
+    @JsonIgnore
     DocumentContent content();
```

Comment:
Our support and our implementation teams rely heavily on this content property. They use it for debugging purpose. Perhaps we should check if we have a proper alternative for them, before we remove it.

Reply:
@Klaas-Ritense Where does the frontend expect the property? I've already removed the custom case header in a frontend PR, and I have on in draft for removing the Angular env-based case list columns.